lib/bugsnag/utility/feature_flag_delegate.rb

```diff
@@ -0,0 +1,77 @@
+module Bugsnag::Utility
+  # @api private
+  class FeatureFlagDelegate
+    def initialize
+      # feature flags are stored internally in a hash of "name" => <FeatureFlag>
+      # we don't use a Set because new feature flags should overwrite old ones
+      # that share a name, but FeatureFlag equality also uses the variant
+      @storage = {}
+    end
+
+    # Add a feature flag with the given name & variant
+    #
+    # @param name [String]
+    # @param variant [String, nil]
+    # @return [void]
+    def add(name, variant)
+      @storage[name] = Bugsnag::FeatureFlag.new(name, variant)
```

Comment:
potentially silly thing but if `nil` is passed as the name, it will be used as the key and may be a problem later.

Reply:
Sounds good :+1: